src/EFTools/EntityDesign/CodeGeneration/Generators/TextTemplatingHost.cs

```diff
@@ -109,6 +110,7 @@ public ITextTemplatingSession CreateSession()
             return new TextTemplatingSession();
         }
 
+        [SuppressMessage("Microsoft.Reliability", "CA2000:Dispose objects before losing scope")]
```

Comment:
Curious, what object is disposable here? Engine? Asking because the warning may be flagging an actual bug in our code.

Reply:
Yes, `Engine`. That code has been like that for multiple major versions - so it's stable and I'd rather not destabilize it while trying to fix this urgent issue. Plus we have many suppressions like this - so it's a more general task to go through the code and remove them.